code/controllers/subsystem/dbcore.dm

```diff
@@ -46,14 +127,27 @@ SUBSYSTEM_DEF(dbcore)
 			"UPDATE [format_table_name("round")] SET shutdown_datetime = Now(), end_state = :end_state WHERE id = :round_id",
 			list("end_state" = SSticker.end_state, "round_id" = GLOB.round_id)
 		)
-		query_round_shutdown.Execute()
+		query_round_shutdown.Execute(force = TRUE)
 		qdel(query_round_shutdown)
 	if(IsConnected())
 		Disconnect()
 
 //nu
 /datum/controller/subsystem/dbcore/can_vv_get(var_name)
-	return var_name != NAMEOF(src, connection) && var_name != NAMEOF(src, active_queries) && ..()
+	if(var_name == NAMEOF(src, connection))
+		return FALSE
+	if(var_name == NAMEOF(src, all_queries))
+		return FALSE
+	if(var_name == NAMEOF(src, queries_active))
+		return FALSE
+	if(var_name == NAMEOF(src, queries_new))
+		return FALSE
+	if(var_name == NAMEOF(src, queries_standby))
+		return FALSE
+	if(var_name == NAMEOF(src, queries_active))
+		return FALSE
+
+	return ..()
 
 /datum/controller/subsystem/dbcore/vv_edit_var(var_name, var_value)
```

Comment:
I do not think vv_edit_var checks that the var was allowed to be gotten, and could still be edited by spoofing hrefs.

You likely need to add the same code down here.